pkg/secrets/info_nix.go

```diff
@@ -0,0 +1,45 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build !windows
+
+package secrets
+
+import (
+	"fmt"
+	"os/user"
+	"strconv"
+	"syscall"
+)
+
+func (info *SecretInfo) populateRights() {
+	err := checkRights(secretBackendCommand)
```

Comment:
Should this be `err := checkRights(info.ExecutablePath)` ?  (I know they're actually the same thing in this case, but would it be semantically more correct?)

Reply:
:+1: